lib/decompress/zstd_decompress_internal.h

```diff
@@ -106,6 +106,8 @@ typedef struct {
     size_t ddictPtrCount;
 } ZSTD_DDictHashSet;
 
+#define ZSTD_LITBUFFEREXTRASIZE    8192 /* extra buffer reduces amount of dst required to store litBuffer */
```

Comment:
8 KB is a significant budget.
Ideally, I would have preferred this permanent internal buffer to be reduced to something like 512 bytes (or less).
But hey, if you state that this large size is necessary otherwise it impacts performance, there is indeed a case for it.

Reply:
I don't know that it is necessary, bumping it up was enough to get several cases that otherwise wouldn't have to run entirely within the buffer, but it's not 100% clear that that is a requirement to get adequate performance.